package.json

```diff
@@ -21,6 +21,7 @@
   "devDependencies": {
     "chai": "^4.1.2",
     "husky": "^1.1.0",
+    "jest": "^23.6.0",
```

Comment:
We are using chai and mocha to test this project, so we do not need Jest.

Reply:
Removed